src/Relation/Binary/Bundles.agda

```diff
@@ -338,18 +357,3 @@ record ApartnessRelation c ℓ₁ ℓ₂ : Set (suc (c ⊔ ℓ₁ ⊔ ℓ₂)) w
     isApartnessRelation : IsApartnessRelation _≈_ _#_
 
   open IsApartnessRelation isApartnessRelation public
-
-
-
-
-------------------------------------------------------------------------
--- DEPRECATED
-------------------------------------------------------------------------
--- Please use the new names as continuing support for the old names is
--- not guaranteed.
-
--- Version 2.0
-
-{-# WARNING_ON_USAGE Preorder._∼_
-"Warning: Preorder._∼_ was deprecated in v2.0. Please use Preorder._≲_ instead. "
-#-}
```

Comment:
Not sure I quite understand why this particular deprecation warning has been removed. Is this removal part of this PR, or a knock-on consequence of (a refactoring of) #2099 ?

Reply:
Oh, OK, I see the warning has now moved to L105-L108. Fine!

Reply:
Ah sorry yes. I should have commented, and I guess add to the style guide, that deprecation warnings should be attached to the definition of the name. Ideally at the end of the file, but in the case of a record declaration at the end of the record.

The problem with putting it at the end, is that the warning wouldn't trigger if the deprecated definition was used between the declaration and the end of the file!